test/bin/build_rpms.sh

```diff
@@ -14,7 +14,7 @@ rm -rf _output/rpmbuild*
 
 BUILD_CMDS=( \
     # Normal build of current branch from source
-    'make rpm' \
+    'make rpm-podman' \
```

Comment:
This add new requirement to test harness. I think we should mention it in the docs

Reply:
yes actually its requires:
1. pull-secret for the registry.ci.openshift.org 
2.  build has to run from a RHEL  RHSM registered machine

Reply:
FYI you could also copy entitlement from RHEL to Fedora and it should work. But for docs, let's keep it simple

Reply:
Can/should we make this optional, only if registry.ci.openshift.org pull secret is present?

Reply:
@ggiguash , i can do something different:
if the command `make rpm-podman` then fallback to `make rpm`

Reply:
@pmtk  , now this change affect CI  only its not really affects the dev-env requirement. do we still need to change the Docs?

Reply:
Nah, I think this is fine